tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -214,6 +249,7 @@ def test_lm_featurizer_shape_values(
         assert computed_sentence_vec.shape[1] == expected_shape[index][1]
 
         # Look at the value of first dimension for a few starting timesteps
+        """
```

Comment:
This part should be uncommented I guess?

Reply:
Yes 🤦‍♀️